lexicon.html

```diff
@@ -130,6 +136,10 @@ <h2>Eve Terms</h2>
       other. Approximately 30,000 km across. Things that are "off grid" may
       show up on <a href="#d-scan">d-scan</a> but not on overview.
     </p>
+    <p>
```

Comment:
this is already listed as an acronym in the bottom section I think, or if not it should be